docs/src/main/paradox/release-notes.md

```diff
@@ -8,6 +8,9 @@
 * _Breaking_: `rf_spatial_index` renamed `rf_xz2_index` to differentiate between XZ2 and Z2 variants.
 * Added `withSpatialIndex` to RasterSourceDataSource to pre-partition tiles based on tile extents mapped to a Z2 space-filling curve 
 * Add `rf_mask_by_bit`, `rf_mask_by_bits` and `rf_local_extract_bits` to deal with bit packed quality masks. Updated the masking documentation to demonstrate the use of these functions.
+* Added `toDF` extension method to `MultibandGeoTiff`
+* Added `rf_agg_extent` and `rf_agg_reprojected_extent` to compute the aggregate extent of a column
```

Comment:
`rf_agg_reprojected_extent` is not documented in the reference.md page. Is it intended to be user facing?